docs/source/user-guide/vision/quickstarts/plot_segmentation_tutorial.py

```diff
@@ -27,151 +27,108 @@
 # %%
 # Defining the data and model
 # ===========================
+# .. note::
+#   In this tutorial, we use the pytorch to create the dataset and model. To see how this can be done using tensorflow
+#   or other frameworks, please visit the :ref:`creating VisionData guide <vision_data__creating_vision_data>`.
 
-# Importing the required packages
-import matplotlib.pyplot as plt
-import numpy as np
-import torch
-import torchvision.transforms.functional as F
-
-
-# %%
```

Comment:
Keep this `# %%`, otherwise the following text is rendered as a comment rather than rst text